package polar_express;

// Next tag: 6
message GlacierVaultDescription {
  optional string creation_date = 1;
  optional string last_inventory_date = 2;
  optional int64 number_of_archives = 3;
  optional string vault_arn = 4;
  optional string vault_name = 5;
}

// Next tag: 3
message GlacierVaultList {
  optional string marker = 1;
  repeated GlacierVaultDescription vault_descriptions = 2;
}
